src/app/app.module.ts

```diff
@@ -21,19 +22,20 @@ import { NotFoundComponent } from './components/not-found/not-found.component';
 import { LoginComponent } from './components/operations/login/login.component';
 import { RegisterComponent } from './components/operations/register/register.component';
 import { MyItinerariesComponent } from './components/itinerary/my-itineraries/my-itineraries.component';
-import { ItineraryViewContoller } from './components/itinerary/itineraryshow/itineraryview.component';
+import { ProfileComponent } from './components/profile/profile.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7182177)
```suggestion
import { ProfileComponent } from "./components/profile/profile.component";
```